Resources/bin/build.sh

```diff
@@ -49,7 +49,7 @@ fi
 cd /tmp/Symfony
 
 # cleanup
-rm -rf app/cache/* app/logs/* .git*
+rm -rf app/cache/* app/logs/* .git/
```

Comment:
the right fix would be to preserve the gitignore file, as done when building the demo today already